addons/binding/org.openhab.binding.venstarthermostat/ESH-INF/thing/thing-types.xml

```diff
@@ -37,13 +37,13 @@
 	        <parameter name="refresh" type="integer" min="1">
 	            <label>Refresh interval</label>
 	            <description>Specifies the refresh interval in seconds.</description>
-	            <default>60</default>
+	            <default>30</default>
 	        </parameter>
         </config-description>
     </thing-type>
 
     <channel-type id="systemMode">
-        <item-type>String</item-type>
```

Comment:
So i changed this back to a number, I'm not clear why it would be a string if the values are 0-4 ?

Reply:
I made these strings because the value mapping doesn't work when the channel type is number. Not sure if that would be considered a bug in OpenHAB or not...

Reply:
Can you elaborate ?  On my system this works fine from the classic UI, paper UI, mobile and Habpanel.   For reference my items look like
```
 Group  F1_GreatRoom_HVAC                        "Great Room Thermostat"                                                                          {alexa="Thermostat"}
    Number F1_GreatRoom_HVAC_Temperature            "Temperature [%.1f° F]"      <dd_thermostat>	    (F1_GreatRoom_HVAC,HVAC_Temperatures_F1)     {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:temperature", alexa="TemperatureSensor.temperature" [scale="Fahrenheit"] }
    Number F1_GreatRoom_HVAC_HeatSetpoint           "Heat Setpoint [%.0f° F]"    <dd_thermostat_heat>	(F1_GreatRoom_HVAC)		                     {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:heatingSetpoint", alexa="ThermostatController.upperSetpoint" [scale="Fahrenheit"] }
    Number F1_GreatRoom_HVAC_CoolSetpoint           "Cool Setpoint [%.0f° F]"    <dd_thermostat_cool>	(F1_GreatRoom_HVAC)		                     {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:coolingSetpoint", alexa="ThermostatController.lowerSetpoint,ThermostatController.tagetSetpoint" [scale="Fahrenheit"]}
    Number F1_GreatRoom_HVAC_Mode                   "Mode [%s]"                 	                    (F1_GreatRoom_HVAC,HVAC_Mode)		         {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:systemMode",  alexa="ThermostatController.thermostatMode" [OFF=0,HEAT=1,COOL=2,AUTO=3]}
    Number F1_GreatRoom_HVAC_Humidity               "Humidity [%d %%]"                          	    (F1_GreatRoom_HVAC)                          {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:humidity"}
    Number F1_GreatRoom_HVAC_State                  "State [%s]"                	                    (F1_GreatRoom_HVAC)                          {channel="venstarthermostat:colorTouchThermostat:a0cc2b50e542:systemState"}
```